src/plugins/intel_cpu/tests/functional/CMakeLists.txt

```diff
@@ -26,7 +26,10 @@ endif()
 addIeTargetTest(
         NAME ${TARGET_NAME}
         ROOT ${CMAKE_CURRENT_SOURCE_DIR}
-        INCLUDES ${INCLUDES}
+        INCLUDES
+            ${INCLUDES}
+            ${CMAKE_SOURCE_DIR}/src/tests/functional/plugin/shared/include
```

Comment:
why did you make such changes in cmake? Common and shared are linked to the binary files

Reply:
Fixed.